examples/scala/src/main/scala/example/Streaming.scala

```diff
@@ -45,14 +46,17 @@ object Streaming {
     val df = spark.read.format("delta").load(path)
     df.show()
 
+
+    println("=== Section 2: write and read delta stream")
     println("Streaming write")
     val streamingDf = spark.readStream.format("rate").load()
     val tablePath2 = new File("/tmp/delta-table2").getCanonicalPath
+    val checkpointPath = new File("/tmp/checkpoint").getCanonicalPath
     val stream = streamingDf
       .select($"value" as "id")
       .writeStream
       .format("delta")
-      .option("checkpointLocation", new File("/tmp/checkpoint").getCanonicalPath)
+      .option("checkpointLocation", checkpointPath)
       .start(tablePath2)
 
     stream.awaitTermination(10000)
```

Comment:
Can remove the println just below this now.

Reply:
Yes, I removed the line println("Reading from stream")